.gitignore

```diff
@@ -39,6 +39,7 @@
 **/build/
 **/cmake-build-debug/
 **/install/
+CMakeFiles/
```

Comment:
Do you call cmake in bezman root?

Reply:
no but if you did you would have this folder here and just to make sure it is ignored I added it to the .gitignore